src/lib.rs

```diff
@@ -1227,9 +1227,41 @@ pub struct TextDocumentClientCapabilities {
 #[derive(Debug, PartialEq, Clone, Default, Deserialize, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub struct WindowClientCapabilities {
-    /// Whether client supports create a work done progress UI from the server side.
+    /// Whether client supports handling progress notifications. If set
+    /// servers are allowed to report in `workDoneProgress` property in the
+    /// request specific server capabilities.
+    ///
+    /// @since 3.15.0
     #[serde(skip_serializing_if = "Option::is_none")]
     pub work_done_progress: Option<bool>,
+
+    /// Capabilities specific to the showMessage request
+    ///
+    /// @since 3.16.0 - proposed state
+    ///
+    #[cfg(feature = "proposed")]
+    pub show_message: ShowMessageRequestClientCapabilities,
```

Comment:
Shouldn't this be an Option?

Reply:
Good question. I've filed https://github.com/microsoft/language-server-protocol/issues/1133 to track.

Reply:
This has been fixed upstream. Good catch.